lib/plugins/aws/lib/naming.js

```diff
@@ -242,6 +242,13 @@ module.exports = {
   getMethodLogicalId(resourceId, methodName) {
     return `ApiGatewayMethod${resourceId}${this.normalizeMethodName(methodName)}`;
   },
+  getValidatorLogicalId(resourceId, methodName) {
+    return `ApiGatewayMethod${resourceId}${this.normalizeMethodName(methodName)}Validator`;
```

Comment:
Super nitpicky but this can be futher simplified via `this.getMethodlogicalId(resourceId, '')` (if I'm not mistaken)